feelpp/feel/feelcore/environment.cpp

```diff
@@ -1410,7 +1410,11 @@ Environment::doOptions( int argc, char** argv,
         std::reverse(configFiles.begin(),configFiles.end());
         for ( std::string const& cfgfile : configFiles )
         {
-            if ( !fs::exists( cfgfile ) ) continue;
+            if ( !fs::exists( cfgfile ) )
+            {
```

Comment:
not good because there is by default config-file name. So all app which not requires a cfg will print a warning (a lot of app of the testsuite for example).

Reply:
I do not really fan about to have default cfg. I think also options config-file and config-files should be gathered.
In this case, we don't manipulate a string but a vector of string. I think it's cleaner to have an empty vector than an empty string.

Reply:
The default config-file name should be empty, no ? So we know that if there is one or more config files, it's the user that entered them and they should be valid.
We could check for `appname+".cfg"` if no config file is found to keep the compatibility with previous versions.

Reply:
how about adding an option that check/ensure the existence of the configfile. by default it would be off (to follow current behavior) but would allow user to crash the app if the config files are not existing/found ?

Reply:
I think that the solution I proposed should not break existing codes.
- If a user do not change the config file name, it checks the `appname.cfg` and if not found continue without configuration (as it happens currently).
- If a user gives a config file name, this file should exists otherwise the app gives a clear warning, or even crash.

@Trophime had this kind of error today, the config file was not read and there is no clear indication of that in the logs or in the output.

Reply:
I'm fine with @prudhomm proposition. 
That could be used as a way to check the consistency of cfg and more generally input data
which is **REALLY** needed

Reply:
@prudhomm I think the current behavior (search a default cfg or mesh) is "contre intuitif".  It's a feeling shared by some users. We can add an option (--demo for example) which allow to use a default cfg.

Reply:
you mean if the application is called `a` we look for `a.cfg` is that it ?

Reply:
yes

Reply:
but it is done only if no cfg is provided, no ?

Reply:
maybe the mesh is more problematic and can do weird stuff. It can easily be disabled

Reply:
the idea is to merge config-file and config-files